tests/functional/aws-node-sdk/test/bucket/deleteBucketTagging.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+const async = require('async');
+
+const getConfig = require('../support/config');
+
+const bucket = 'policyputtaggingtestbucket';
+
+const validTagging = {
+    TagSet: [
+        {
+            Key: 'key1',
+            Value: 'value1',
+        },
+        {
+            Key: 'key2',
+            Value: 'value2',
+        },
+    ],
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be  ' +
+            `${errors[expectedErr].code}, but got '${err.statusCode}'`);
+    }
+    cb();
+}
+
+describe('aws-sdk test delete bucket tagging', () => {
+    let s3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        return done();
+    });
+
+    beforeEach(done => s3.createBucket({ Bucket: bucket }, done));
+
+    afterEach(done => s3.deleteBucket({ Bucket: bucket }, done));
+
+    it('should delete tag', done => {
+        async.series([
```

Comment:
we might need to update to v3 of the aws sdk but we should at some point and use async await for this sort of stuff.

Reply:
Yea I was just mentioning it cause we should use async await but in v2 its a bit harder.